### ISSUE TITLE
[BUG] - Anchor tags inside opened accordion if used in a flyout are focusable/tabbable though it's hidden

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2799]
### Bug description
If the `accordion` component with an anchor link is used as a slot in the `flyout` component, this anchor link is still focusable in the following scenario:

- `flyout`is hidden
- `accordion` has state `open`

This only occurs on links slotted inside an `accordion`. If just an anchor tag is provided as a slot inside the `flyout` it doesn't get focused. Also the `accordion` itself doesn't get focused, though it's a focusable element.

### JS Framework

all

### Porsche Design System version

3.7.0

### What browsers are you seeing the problem on?

all

### Browser version

latest

### Notes
- `inert` attribute (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert) might be a solution to solve the issue 

### Reproduction Example

https://stackblitz.com/edit/twyku1?file=index.html

### Steps to reproduce the behavior

1. Go to Stackblitz example
2. Tab the links and look at the missing focus and the link url preview in the bottom of your browser
3. You will see, that the anchor link inside the flyout > accordion gets focused even when the flyout is hidden

### Expected behavior

All interactive element inside a flyout are not focusable if the flyout is hidden / `<p-flyout open="false" …`.
